src/app/projects/dockable-tab-project.view.ts

```diff
@@ -702,6 +702,13 @@ export class SectionUnloadedProjects extends Section {
                         `Projects fails (${failures.length})`,
                 },
                 icon: 'fas fa-exclamation-triangle',
+                handler: true,
```

Comment:
here instead of `projectState` & `onclick` & `handler` you provide the definition of the `expanderView` using the parameters `withChildren`. The observable  `dropdownHandler$` should be declared in this class, it will be used by the `expanderView` (the icon with `onclick` callback), and wrap the `content` section to display it or not.